src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -296,5 +300,32 @@ private static NullableAnnotation GetNullableAnnotation(this BoundExpression exp
 
             return NullableAnnotation.Unknown;
         }
+
+        public static BoundKind KindIgnoringSuppressions(this BoundExpression expr)
```

Comment:
>KindIgnoringSuppressions [](start = 32, length = 24)

`KindIgnoringSuppressions()` seems a little unnecessary. In particular, there seem to be a number of callers that use `expr.KindIgnoringSuppressions()` followed by `var exprWithoutSuppressions = expr.RemoveSuppressions();`. If we remove this extension method, then those callers might be rewritten as:
```
var exprWithoutSuppressions = expr.RemoveSuppressions();
... exprWithoutSuppressions.Kind ...
```
The remaining callers would use the slightly more verbose: `expr.RemoveSuppressions().Kind`.